cmd/imagec/imagec.go

```diff
@@ -226,7 +230,7 @@ func DestinationDirectory() string {
 }
 
 // ImagesToDownload creates a slice of ImageWithMeta for the images that needs to be downloaded
-func ImagesToDownload(manifest *Manifest, storeName string) ([]*ImageWithMeta, error) {
+func ImagesToDownload(manifest *Manifest, storeName string) ([]*ImageWithMeta, *ImageWithMeta, error) {
```

Comment:
confused, why we are returning an additional ImageWithMeta here? As far as I see it's alwasy the first item on the list so we can drop all the changed in this function and let caller to get first item.
```
eg; 
```
if err := updateImageMetadata(images[0], manifest); err != nil {
```
instead of
```
if err := updateImageMetadata(imageLayer, manifest); err != nil {
```